fantiadl/models.py

```diff
@@ -293,15 +293,15 @@ def download_paid_fanclubs(self, limit=0):
                 else:
                     raise
 
-    def download_new_posts(self, post_limit=24):
+    def download_new_posts(self, post_limit=24, paid_only=False):
         all_new_post_ids = []
         total_pages = math.ceil(post_limit / 24)
         page_number = 1
         has_next = True
-        self.output("Downloading {} new posts...\n".format(post_limit))
+        self.output("Downloading {} new{} posts...\n".format(post_limit, " paid" if paid_only else ""))
 
         while has_next and not len(all_new_post_ids) >= post_limit:
-            response = self.session.get(TIMELINES_API.format(page_number))
+            response = self.session.get(TIMELINES_API.format(page_number, "&free_plan=not_free" if paid_only else ""))
```

Comment:
This should use `urlencode`, or specify `params` with `requests.get`: https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls